test/acceptance/tests/basic/basic_test.go

```diff
@@ -664,148 +772,150 @@ func TestValidation_RolePath(t *testing.T) {
-			lbSubnets: []string{"subnet"},
-			lbVpcID:   "vpc",
-		},
-		"lb_enabled and no lb subnets": {
-			kind:      "mesh-gateway",
-			lbEnabled: true,
-			lbVpcID:   "vpc",
-			expError:  "lb_subnets is required when lb_enabled is true",
-		},
-		"lb_enabled and no VPC": {
-			kind:      "mesh-gateway",
-			lbEnabled: true,
-			lbSubnets: []string{"subnet"},
-			expError:  "lb_vpc_id is required when lb_enabled is true",
-		},
-		"lb create security group and modify security group": {
-			kind:             "mesh-gateway",
-			securityGroups:   []string{"sg"},
-			lbEnabled:        true,
-			lbSubnets:        []string{"subnet"},
-			lbVpcID:          "vpc",
-			lbCreateSecGroup: true,
-			lbModifySecGroup: true,
-			expError:         "Only one of lb_create_security_group or lb_modify_security_group may be true",
-		},
-		"lb modify security group and no security group ID": {
-			kind:               "mesh-gateway",
-			securityGroups:     []string{"sg"},
-			lbEnabled:          true,
-			lbSubnets:          []string{"subnet"},
-			lbVpcID:            "vpc",
-			lbCreateSecGroup:   false,
-			lbModifySecGroup:   true,
-			lbModifySecGroupID: "",
-			expError:           "lb_modify_security_group_id is required when lb_modify_security_group is true",
-		},
-		"lb modify security group with security group ID": {
-			kind:               "mesh-gateway",
-			securityGroups:     []string{"sg"},
-			lbEnabled:          true,
-			lbSubnets:          []string{"subnet"},
-			lbVpcID:            "vpc",
-			lbCreateSecGroup:   false,
-			lbModifySecGroup:   true,
-			lbModifySecGroupID: "mod-sg",
-		},
-	}
-	for name, c := range cases {
-		c := c
-		t.Run(name, func(t *testing.T) {
-			t.Parallel()
-
-			tfVars := map[string]interface{}{
-				"enable_mesh_gateway_wan_federation": c.enableMeshGatewayWANFed,
-				"tls":                                c.tls,
-				"security_groups":                    c.securityGroups,
-				"wan_address":                        c.wanAddress,
-				"lb_enabled":                         c.lbEnabled,
-				"lb_subnets":                         c.lbSubnets,
-				"lb_vpc_id":                          c.lbVpcID,
-				"lb_create_security_group":           c.lbCreateSecGroup,
-				"lb_modify_security_group":           c.lbModifySecGroup,
-				"lb_modify_security_group_id":        c.lbModifySecGroupID,
-			}
-			if len(c.kind) > 0 {
-				tfVars["kind"] = c.kind
-			}
-			applyOpts := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
-				TerraformDir: terraformOptions.TerraformDir,
-				NoColor:      terraformOptions.NoColor,
-				Vars:         tfVars,
-			})
-			t.Cleanup(func() { _, _ = terraform.DestroyE(t, applyOpts) })
-
-			_, err := terraform.PlanE(t, applyOpts)
-			if len(c.expError) > 0 {
-				require.Error(t, err)
-				require.Contains(t, err.Error(), c.expError)
-			} else {
-				require.NoError(t, err)
-			}
-		})
-	}
-}
+// TODO: Revisit this test
+//
+// func TestValidation_MeshGateway(t *testing.T) {
```

Comment:
nit / suggestion: You can add a `t.Skip` instead of commenting out the test if you prefer: `t.Skip("TODO(ganeshrocks): revisit this test for agentless")`. Then you'll see the skipped test in the test run output.